resource/mail/acc32_mail_secret2_english.txt

```diff
@@ -1,7 +1,7 @@
 GAME
 {
    	"MailFrom1"	"From: DJH"
-	"MailDate1"	"Date: July 6th, 2022"
+	"MailDate1"	"Date: July 6th, 2222"
```

Comment:
This one was canonically written 4 days ago in real life (as of this comment), so the year doesn't need to be changed.